### DISCUSSION TITLE
The absolute max/min for phi in max-camera-orbit and min-camera-orbit is undocumented, and overrides user input

### DISCUSSION THREAD

**Discussion by daniel-packard** [google/model-viewer#3219] [Q&A]
### Description
There is an undocumented absolute maximum that overrides user input for `phi` in `max-camera-orbit` and `min-camera-orbit`- and it's not possible to extend rotation limits in the `phi` direction past the absolute max/min of +/- 180deg. 

e.g. ...
```
  max-camera-orbit="auto 360deg auto"
```
...caps `phi` at 180deg, not 360deg as requested.

This limit feels awkward, especially for any model that is not oriented to the default coordinate system. 

Request to either: 
a) document the absolute maximum limit for `phi` along with rationale, or 
b) remove/extend the max limit beyond +/- 180deg

#### Live Demo
https://catkin-walnut-gateway.glitch.me

### Version
model-viewer: v1.10.1

**Comment by daniel-packard** [comment]
Here's a better real-world example that illustrates how awkward it can be: 

https://view2.spectre3d.io/f62862e6-8f08-4b7d-a742-142bc1c3745a

**Reply by elalish** [reply]
Thanks for the example! I'm going to call this by design, though agreed we should call it out in the docs. The better way to solve your problem is using our [orientation](https://modelviewer.dev/docs/index.html#entrydocs-scenegraph-attributes-orientation) attribute.

**Reply by daniel-packard** [reply]
Thanks for the reply!

In our use case (3D scanning), it can be challenging to determine the orientation of the 3D model automatically - but we can probably use your `orientation` property to give the use some control here 🤔 

Conceptually I understand the design decision, in that it doesn't really make sense to allow coordinates outside the standard bounds of polar coordinates for theta/phi. 

But when thinking of positioning a "real camera" in a scene, it doesn't make sense to me that it would be constrained this way (e.g. if I were holding a real camera with my hand, I could easily position it from any side of the object with any orientation - unconstrained by "polar coordinate boundaries")

For comparison, here is a small gif of the native viewer we have coded in swift/metal for our iOS app - it is unconstrained at the poles:
![RPReplay_Final1645562683](https://user-images.githubusercontent.com/1051245/155217579-5e765af6-2872-4a17-bdcd-b0c3c3ac5698.gif)

Totally understand if you prefer to keep it as designed - I'll look into setting the orientation property to improve user experience for our scans

thanks again
/d
